specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -287,12 +287,12 @@ interoperability protocol based on RMI-IIOP.
 ==== Support for Transactional Resource Managers
 
 This specification requires all Jakarta EE products to support the
-_jakarta.transaction.xa.XAResource_ interface, as specified in the Connector
+_javax.transaction.xa.XAResource_ interface, as specified in the Connector
```

Comment:
The XAResource interface does not define the javax.transaction.xa.XAResource interface, the Java binding is provided by Java SE. The reference in "Connectors" (please note the plural which depending on the choice here should still be altered): https://jakarta.ee/specifications/connectors/2.0/connectors-spec-2.0.html#interface-xaresource can be helpful but I think it is more referenced in the Connectors specification rather than specified by it even then /cc @jmstephensgit @smillidge

Reply:
Good point, @tomjenkinson.  Thanks!  A simple change would be to change this to "..as referenced in the Connectors specification."  And, I do agree that we should update this to "Connectors" (vs "Connector").  Would this update satisfy your comment?

Reply:
It would - thanks!

Reply:
Done

Reply:
Thanks!